src/monocle_apptrace/wrap_common.py

```diff
@@ -266,3 +269,28 @@ def update_tags(instance, span):
         span.set_attribute(TAGS, [model_name, vector_store_name])
     except:
         pass
+
+
+def update_attributes(instance, span):
+    """
+       Updates the telemetry span attributes for vector store retrieval tasks.
+    """
+    try:
+        # Check if the span is for a vector store retriever task from langchain
+        if span.name == 'langchain.task.VectorStoreRetriever':
```

Comment:
span.name is not consistent across llama index and haystack.
Please check and handle the cases

Reply:
done